packages/svrx-util/__tests__/spec/package-manager.test.js

```diff
@@ -57,25 +133,31 @@ describe('Package Manager', () => {
       const plugin = await pm.load();
       expect(plugin.name).to.eql('hello');
       expect(plugin.version).to.eql('1.0.1');
-      expect(plugin.path).to.eql(libPath.join(TEST_SVRX_DIR, 'plugins/hello/1.0.1'));
+      expect(plugin.path)
+        .to
+        .eql(libPath.join(TEST_SVRX_DIR, 'plugins/hello/1.0.1'));
     });
     it('should work fine when load with a remote version and auto load a latest version', async () => {
-      const storePath = libPath.join(TEST_SVRX_DIR, 'plugins/demo/1.0.2');
-      const storePathLatest = libPath.join(TEST_SVRX_DIR, 'plugins/demo/1.0.3');
-      after(async () => {
-        await cleanModule(storePath);
-        await cleanModule(storePathLatest);
-      });
       const pm = PackageManagerCreator({
         plugin: 'demo',
         coreVersion: '0.0.2',
         version: '1.0.2',
       });
+
+      const storePath = libPath.join(TEST_SVRX_DIR, 'plugins/demo/1.0.2');
+      const storePathLatest = libPath.join(TEST_SVRX_DIR, 'plugins/demo/1.0.3');
+      after(async () => {
+        await pm.remove('demo/1.0.2');
+        await pm.remove('demo/1.0.3');
+      });
+
       const plugin = await pm.load();
       expect(plugin.name).to.eql('demo');
       expect(plugin.version).to.eql('1.0.2');
       expect(plugin.path).to.eql(storePath);
-      expect(fs.existsSync(libPath.join(storePathLatest, 'index.js'))).to.eql(true);
+      expect(fs.existsSync(libPath.join(storePathLatest, 'index.js')))
```

Comment:
eql means `loose(deep) equal`,  use `equal` instead

Reply:
Since the values are mostly strings and booleans, I think both `eql` and `equal` will do?